web/react-web/src/Sidebar/index.js

```diff
@@ -0,0 +1,113 @@
+import React, {Component} from 'react';
+import {AvailablePredictionsContext, GraphDetailContext, MODE_LINEAR, MODE_LOG, MODE_LOG_LOG} from "../sharedObjects";
+import CountriesWrapper from "./CountriesWrapper";
+
+const options = { year: 'numeric', month: 'long', day: 'numeric' };
+
+class Sidebar extends Component {
+    state = {
+        activeTab: 0
+    };
+
+    selectTab = (activeTab) => () => {
+        this.setState({activeTab})
+    };
+
+    changeGraphDetail = (mode, relative) => () => {
+        const {setOptions} = this.context;
+        if (!relative && mode === MODE_LOG_LOG) {
+            mode = MODE_LINEAR;
+        }
+        setOptions({mode, relative});
+    };
+
+    render() {
+        return (
+            <AvailablePredictionsContext.Consumer>
+                {context =>
+                    <GraphDetailContext.Consumer>
+                        {graphDetailContext => {
+                            const dates = [];
+                            const {relative, mode} = graphDetailContext.options;
+                            const {activeTab} = this.state;
+
+                            for (let key in context.data) {
+                                const item = context.data[key];
+                                const date = new Date(Date.parse(item.label));
+                                const className = `tab${this.state.activeDateKey === key ? ' active' : ''}`;
+
+                                dates.push(<button key={key} className={className} onClick={this.selectTab(key)}>{date.toLocaleDateString(undefined, options)}</button>);
+                            }
+
+                            return (
+                                <>
+                                    <div className='buttons'>
+                                        <button className={activeTab === 0 ? 'active' : ''} onClick={this.selectTab(0)}>About</button>
+                                        <button className={activeTab === 1 ? 'active' : ''} onClick={this.selectTab(1)}>Results</button>
+                                    </div>
+                                    {activeTab === 0 &&
+                                        <>
+                                            <h1>COVID-19 predictions of Boďová and Kollár</h1>
```

Comment:
Please add a TODO that this text should be generated from Markdown that's shared with Python and we will fix it in another PR. Actually, we might need to serve it from Python via REST, as the text might be modified per page.

Reply:
Added.